charts/rancher-csp-adapter/0.1.0+up0.1.0-rc2/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: 0.1.0-rc2
+  catalog.cattle.io/namespace: cattle-csp-system
```

Comment:
```suggestion
  catalog.cattle.io/namespace: cattle-csp-adapter-system
```

Reply:
This would not be a good change to make - Rancher looks for this in cattle-csp-system and that's the namespace that we added to the registered namespaces in rancher. Changing this would require an additional rancher PR so I would prefer to leave as is.

Reply:
IMO just using the word CSP is too broad. It could mean a lot of different things, and sounds more like a csp infrastructure generator or a cloud provider tool. CSP-adapter is at least somewhat more specific, even though it should have been something like `csp-licensing-adapter` or `csp-support-adapter`.